### PR TITLE
Stellar integration

### PR DESCRIPTION
**Pull request by Rakeally**

### CODE REVIEW

**Comment by jayluxferro** [comment]
Hi @Rakeally , I think there's no need for `createAssetAccount` since all accounts (including issuer) can be created with the `generateAddress`.

`ChangeTrust` should be renamed `CreateAsset`.

`sendAsset` should be renamed `SendAsset`.

**Comment by baahkusi** [comment]
We should expose these three operations and the commands should be named as such.

- ChangeTrust
- CreateAccount
- Payment 

for payment operation it should cover for all assets including the native asset so we don't have two separate commands.

File: android/src/main/kotlin/africa/ejara/trustdart/coins/XLM.kt [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        path: String,
+        txData: Map<String, Any>,
+        mnemonic: String,
+        pass_phrase: String
+    ): String? {
+        val cmd = txData["cmd"] as String
+        val secretKey = HDWallet(mnemonic, pass_phrase).getKey(coinType, path)
+        val txHash: String?
+        when (cmd) {
+            "CreateAsset" -> {
```
Comment (baahkusi) [inline android/src/main/kotlin/africa/ejara/trustdart/coins/XLM.kt:43]: change name to `ChangeTrust`

File: android/src/main/kotlin/africa/ejara/trustdart/coins/XLM.kt [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                    fee = txData["fee"] as Int
+                    sequence = (txData["sequence"] as Long).toLong()
+                    passphrase = StellarPassphrase.STELLAR.toString()
+                    opChangeTrust = operation.build()
+                    privateKey = ByteString.copyFrom(secretKey.data())
+                }
+                val output = AnySigner.sign(signingInput.build(), coinType, SigningOutput.parser())
+                txHash= output.signature
+            }
+            "SendAsset" -> {
```
Comment (baahkusi) [inline android/src/main/kotlin/africa/ejara/trustdart/coins/XLM.kt:66]: change name to `Payment`

File: android/src/main/kotlin/africa/ejara/trustdart/coins/XLM.kt [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                    sequence = (txData["sequence"] as Long).toLong()
+                    passphrase = StellarPassphrase.STELLAR.toString()
+                    opPayment = operation.build()
+                    privateKey = ByteString.copyFrom(secretKey.data())
+                }
+                val output = AnySigner.sign(signingInput.build(), coinType, SigningOutput.parser())
+                txHash = output.signature
+
+            }
+            "XLM" -> {
```
Comment (baahkusi) [inline android/src/main/kotlin/africa/ejara/trustdart/coins/XLM.kt:91]: merge this one with the above

**Review by baahkusi — CHANGES_REQUESTED** [review]

**Review by baahkusi — APPROVED** [review]

**Pull request merged**